README.md

```diff
@@ -1,64 +1,72 @@
-Dash Player: Control and play videos using Dash
+# Dash Player
 
 ![GitHub](https://img.shields.io/github/license/mashape/apistatus.svg)
 [![GitHub stars](https://img.shields.io/github/stars/xhlulu/dash-player.svg)](https://github.com/xhlulu/dash-player/stargazers)
 
-Dash Player is a dash component for playing a variety of URLs, including file paths, YouTube, Facebook, Twitch, SoundCloud, Streamable, Vimeo, Wistia, Mixcloud, and DailyMotion. It is wrapped around the react-player component.
+Dash Player is a Dash component for playing a variety of URLs, including file paths, YouTube, Facebook, Twitch, SoundCloud, Streamable, Vimeo, Wistia, Mixcloud, and DailyMotion. It is wrapped around the [ReactPlayer](https://github.com/cookpete/react-player) component.
 
-You can find a [demo of the component here](http://dash-player-usage.herokuapp.com).
-
-For updates and more, please see the dash community discussion on Dash Player.
+For updates and more, please see the ongoing changes to this repository's issue tracker or the Dash community discussion on Dash Player.
 
 This is a custom community component, so if your organization or company is interested in sponsoring enhancements to this project, [please reach out to the Plotly Dash team](https://plot.ly/dash/pricing).
 
 ## Getting started
 
-Here are the following steps to get started with your own apps
+Here are the following steps to get started with using Dash Player in your own Dash apps
 
 ```sh
-$ git clone https://github.com/xhlulu/dash-player.git
+$ git clone https://github.com/plotly/dash-player
 $ cd dash-player
 $ npm install
-$ python usage-advanced.py
 ```
 
-Once that done, you can copy the `dash_player` package in the folder of your app, and import it within your app.
+Once that's done, you can copy the `dash_player` package in the folder of your app, and import it within your app.
+
+## Build manually
+
+To generate builds manually, run the following:
+
+```sh
+$ npm install webpack webpack-cli
+$ npm run build:all
```

Comment:
FWIW in other packages we've simplified the "full build" command to just `npm run build`

Reply:
Will change in #2